cmd/openshift-install/create.go

```diff
@@ -427,6 +427,9 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *cluster
 	timer.StartTimer("API")
 
 	if assetStore, err := assetstore.NewStore(command.RootOpts.Dir); err == nil {
+		if vErr := assetStore.Validate(); vErr != nil {
+			return newClientError(vErr)
+		}
```

Comment:
Can we validate the certs in the same manner as we do with other assets? For example, add the cert expiration check to the `PlatformProvisionCheck` or adding a new validation in the `ignition-configs` stage? 

Adding `Validate` to the asset store seems to break the abstraction, ATM the asset store has no knowledge of the contents of the assets it stores. The existing patterns seem like they would be adequate to handle this validation (although they may need to be extended a little).

Reply:
If we move this to `PlatformProvisionCheck` how do we expect to get the certs? Should we expose the asset information in the store? We could attempt to read the state file directly, but each entry is translated into raw messages that are assets.

Reply:
I looked into this further, and it should be even simpler, this is all already handled by the exisitng warnIfCertsExpired function:

https://github.com/openshift/installer/blob/71a467e27d0a681029dcc62b4469f6969ba7550b/pkg/asset/ignition/bootstrap/common.go#L712

We simply need to update that function to throw an error rather than log a warning. 

The asset is loaded during both `create ignition-configs` and `create cluster` so this should handle all scenarios.

Reply:
```
ERROR Bootstrap Ignition-Config Certificate admin-kubeconfig-ca-bundle.crt expired at 2025-05-01T17:40:18Z. 
ERROR Bootstrap Ignition-Config Certificate kube-apiserver-complete-client-ca-bundle.crt expired at 2025-05-01T17:40:18Z. 
FATAL failed to fetch Metadata: failed to load asset "Bootstrap Ignition Config": 2 certificates expired
```

Reply:
we probably should tell them to regenerate the ignition configs in a new directory.